src/Event.tsx

```diff
@@ -28,24 +28,50 @@ interface EventProps {
 }
 
 interface EventState {
-  event?: core.Event
+  event?: core.Event,
+  userHasLogAccess: boolean
 }
 
 // TODO: Need to make this component auto-refresh
 class Event extends React.Component<EventProps, EventState> {
   constructor(props: EventProps) {
     super(props)
-    this.state = {}
+    this.state = {
+      userHasLogAccess: false
+    }
   }
 
   async componentDidMount(): Promise<void> {
+    const coreClient = getClient().core()
+    const eventData = await coreClient.events().get(this.props.id)
+    const projectId = eventData.projectID
+
     this.setState({
-      event: await getClient().core().events().get(this.props.id)
+      event: eventData
     })
+
+    if(projectId) {
+      const authzList = await coreClient.projects().authz().roleAssignments().list(projectId)
```

Comment:
I still recommend passing a selector ad a second argument here.

Reply:
Will be there in next commit.